### PR TITLE
SUS-279 in MiniEditor add warning message when leaving page without submit

### PR DESCRIPTION
**Pull request by sqreek**
[SUS-279](https://wikia-inc.atlassian.net/browse/SUS-279)

Add warning/confirmation message when leaving a page without sumbit changes in minieditor

ping @Wikia/sustaining-team

### CODE REVIEW

**Comment by pawelchcki** [comment]
LGTM

**Comment by macbre** [comment]
LGTM

File: extensions/wikia/EditPageLayout/js/plugins/MiniEditor.js [hunk 1]
```diff
@@ -75,6 +75,13 @@
 		},
 
 		editorActivated: function(event) {
+
+			$(window).on('beforeunload.PreventLeaveBeforeSave', function (e) {
```
Comment (wladekb) [inline extensions/wikia/EditPageLayout/js/plugins/MiniEditor.js:79]: Let's make this suffix a bit more MiniEditor specific to prevent name clash

**Comment by wladekb** [comment]
How about my suggestion https://github.com/Wikia/app/pull/10006/files#r56624798 ?

**Pull request merged**